### PR TITLE
Feat/download

### PR DESCRIPTION
**Pull request by zflamig**
Link file name in the Explorer tab to the download file endpoint

### CODE REVIEW

File: src/Explorer/ExplorerTable.jsx [hunk 1]
```diff
@@ -55,7 +56,9 @@ export class ExplorerTableComponent extends Component {
         <TableData c_width={columnWidths[0]}>
           <Link to={`/${file.project_id}`}>{file.project_id}</Link>
         </TableData>
-        <TableData c_width={columnWidths[1]}>{file.name}</TableData>
+        <TableData c_width={columnWidths[1]}>
```
Comment (philloooo) [inline src/Explorer/ExplorerTable.jsx]: we should not show them as link if user doesn't have read-storage permission on this project?

File: src/Explorer/ExplorerComponent.jsx [hunk 2]
```diff
@@ -50,6 +50,7 @@ class ExplorerComponent extends Component {
   mapDataToFile = filesList => filesList.map(
     file => (
       { project_id: file.project_id,
+        uuid: file.id,
```
Comment (philloooo) [inline src/Explorer/ExplorerComponent.jsx]: did？

File: Dockerfile [hunk 3]
```diff
@@ -33,4 +33,8 @@ RUN COMMIT=`git rev-parse HEAD` && echo "export const portalCommit = \"${COMMIT}
     && NODE_ENV=production ./node_modules/.bin/webpack --bail \
     && cp nginx.conf /etc/nginx/conf.d/nginx.conf \
     && rm /etc/nginx/sites-enabled/default
+
+RUN mkdir /etc/nginx/ssl \
```
Comment (philloooo) [inline Dockerfile]: Not for prod?

Comment (zflamig) [reply Dockerfile]: wrong path, good catch.

**Pull request merged**